src/main/java/com/epam/ta/reportportal/core/widget/content/loader/LaunchesComparisonContentLoader.java

```diff
@@ -22,22 +22,84 @@
 package com.epam.ta.reportportal.core.widget.content.loader;
 
 import com.epam.ta.reportportal.commons.querygen.Filter;
+import com.epam.ta.reportportal.commons.validation.BusinessRule;
 import com.epam.ta.reportportal.core.widget.content.LoadContentStrategy;
+import com.epam.ta.reportportal.core.widget.content.WidgetContentUtils;
+import com.epam.ta.reportportal.dao.LaunchRepository;
+import com.epam.ta.reportportal.dao.WidgetContentRepository;
+import com.epam.ta.reportportal.entity.launch.Launch;
+import com.epam.ta.reportportal.entity.widget.ContentField;
 import com.epam.ta.reportportal.entity.widget.WidgetOption;
+import com.epam.ta.reportportal.entity.widget.content.ComparisonStatisticsContent;
+import com.epam.ta.reportportal.exception.ReportPortalException;
+import com.epam.ta.reportportal.ws.model.ErrorType;
+import org.apache.commons.collections.MapUtils;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Predicate;
+
+import static com.epam.ta.reportportal.commons.Predicates.equalTo;
+import static com.epam.ta.reportportal.commons.Predicates.notNull;
+import static com.epam.ta.reportportal.core.widget.content.WidgetContentUtils.GROUP_CONTENT_FIELDS;
+import static java.util.Collections.singletonMap;
 
 /**
  * @author Pavel Bortnik
  */
 @Service
 public class LaunchesComparisonContentLoader implements LoadContentStrategy {
 
+	@Autowired
+	private LaunchRepository launchRepository;
+
+	@Autowired
+	private WidgetContentRepository widgetContentRepository;
+
 	@Override
-	public Map<String, ?> loadContent(List<String> contentFields, Filter filter, Set<WidgetOption> widgetOptions, int limit) {
-		return null;
+	public Map<String, ?> loadContent(Set<ContentField> contentFields, Filter filter, Set<WidgetOption> widgetOptions, int limit) {
+
+		Map<String, List<String>> fields = GROUP_CONTENT_FIELDS.apply(contentFields);
+		validateContentFields(fields);
+
+		Map<String, List<String>> options = WidgetContentUtils.GROUP_WIDGET_OPTIONS.apply(widgetOptions);
+		validateWidgetOptions(options);
+
+		String launchName = options.get(LAUNCH_NAME_FIELD).iterator().next();
+		Launch latestLaunch = launchRepository.findLatestByNameAndFilter(launchName, filter)
+				.orElseThrow(() -> new ReportPortalException(ErrorType.LAUNCH_NOT_FOUND, "No launch with name: " + launchName));
+
+		//TODO change the way of getting previous launch
+		Long latestLaunchId = latestLaunch.getId();
+
+		List<ComparisonStatisticsContent> result = widgetContentRepository.launchesComparisonStatistics(filter, fields, limit,
+				latestLaunchId - 1,
+				latestLaunchId
+		);
+		return singletonMap(RESULT, result);
```

Comment:
let's make this widget with possibility to compare as much launches as i want to.
not only two. so, i see this in the next way. client specifies launch name and limit. ws gives comparison of 'limited' launches by provided name back.